### DISCUSSION TITLE
Adventure map, display information related to a hero when another hero is positioned on a large object [Done]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3293] [Ideas]
In fheroes2, it would be nice to be able to display information related to a hero ("(not visited)", "(already visited)") when another hero is positioned on a large object (an object that occupies several squares and part of which is directly accessible with the mouse cursor).

In fheroes2:

![Large objet info window for hero fh2 01](https://user-images.githubusercontent.com/43583994/116282933-a2e2d700-a78b-11eb-894d-187a2043406e.png)

**Comment by Branikolog** [comment]
Hi, @LeHerosInconnu 
I believe your suggestion is a part of already existing issue: #1281.

**Reply by LeHerosInconnu** [reply]
Hello @Branikolog,

Yes, I thought about adding it to the mines issue, but I wasn't sure if it was handled the same way.

In the original game, the defect reported here is also present, while for the mines it works correctly.

Do you think it would be better to add it in the mines issue?
Or in another thread, because it is not quite the same?
One that can wait for now?

**Reply by Branikolog** [reply]
@LeHerosInconnu 
I believe this issue is all about the object's parts on a world map. @ihhub even referred a #660 issue there. So, you can add your proposal in #1281, so the one, who will work on object description will take into consideration the problem you described here.
Also, we can always create a separate issue later.

**Reply by LeHerosInconnu** [reply]
@Branikolog.

It's done.
For @ihhub you meant @idshibanov. :D

**Reply by Branikolog** [reply]
Thanks, @LeHerosInconnu !
P. S. @ihhub referred issue with explanation from @idshibanov. )

**Comment by LeHerosInconnu** [comment]
This has been implemented.
I close the discussion.
